src/Symfony/Component/Mailer/Transport/AbstractTransportFactory.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Symfony\Component\Mailer\Transport;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
```

Comment:
if you can use the one from Contracts its better

Reply:
I've created separate PR for that: #31956